### ISSUE TITLE
DefaultJaxrsScanner prettyPrint has no effect

### ISSUE THREAD

**Issue by DerEineDa** [swagger-api/swagger-core#1862]
Seen in Swagger 2.2.0.

DefaultJaxrsScanner extends AbstractScanner. Both classes have a boolean field named "prettyPrint".

DefaultJaxrsScanner.setPrettyPrint(boolean) sets the field DefaultJaxrsScanner.prettyPrint, but the method DefaultJaxrsScanner.getPrettyPrint() reads the value of AbstractScanner.prettyPrint.

DefaultJaxrsScanner has a method called "prettyPrint()" to read the value of DefaultJaxrsScanner.prettyPrint, but this method is never called from Swagger Core. 

To sum it up: There is no way to enable pretty printing when using the DefaultJaxrsScanner.

**Comment by webron** [comment]
What's Swagger 2.2.0? There's swagger-codegen 2.2.0, but that's not swagger-core.

**Comment by DerEineDa** [comment]
I am sorry, I mixed up the versions. Of course I meant the latest swagger-core 1.5.9.

**Comment by fehguy** [comment]
This was just fixed, please try the latest
